pcre.go

```diff
@@ -613,6 +602,35 @@ func (re Regexp) ReplaceAllString(in, repl string, flags int) string {
 	return string(re.ReplaceAll([]byte(in), []byte(repl), flags))
 }
 
+// Match holds details about a single successful regex match.
+type Match struct {
+	Finding string // Text that was found.
+	Loc     []int  // Index bounds for location of finding.
+}
+
+// FindAll finds all instances that match the regex.
+func (re Regexp) FindAll(subject string, flags int) []Match {
+	matches := make([]Match, 0)
+	m := re.MatcherString(subject, flags)
+	offset := 0
+	for m.Matches() {
+		matches = append(
+			matches,
+			Match{
+				subject[int(m.ovector[0])+offset : int(m.ovector[1])+offset],
```

Comment:
Fix spacing between the + operator. Should be consistent with the line below.

Reply:
Actually, you should just compute the indices once and re-use the values for both lines.